packages/collector/test/tracing/database/db2/app.js

```diff
@@ -4,6 +4,15 @@
 
 'use strict';
 
+// ibm_db uses fs-extra
+// fs-extra uses graceful-fs
+// nyc is graceful-fs
+// graceful-fs loaded too early and does not get instrumented
+// TODO: is there a better solution?
+Object.keys(require.cache).forEach(key => {
+  if (key.indexOf('fs') !== -1) delete require.cache[key];
+});
```

Comment:
So the problem is that `nyc` uses `spawn-wrap` and `node-preload`.
They run a foreground spawn process and preload required modules. As soon as I comment our this [line](https://github.com/istanbuljs/nyc/blob/v15.1.0/bin/nyc.js#L59), the problem is resolved.

We haven't run in this problem before because non of our instrumentations were affected by the cached preloads.
Started with the support for `fs`.

This is the list of preloaded modules:
```
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/node-preload/preload-path/node-preload.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/node-preload/internal-preload-module.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/node-preload/preload-list.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/node-preload/preload-list-env.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/node-preload/hook-spawn.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/process-on-spawn/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/fromentries/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/node-preload/generate-require.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/node-preload/process-node-options.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/nyc/lib/register-env.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/nyc/lib/wrap.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/nyc/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/caching-transform/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/hasha/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/is-stream/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/make-dir/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/semver/semver.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/write-file-atomic/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/imurmurhash/imurmurhash.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/signal-exit/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/signal-exit/signals.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/is-typedarray/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/typedarray-to-buffer/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/package-hash/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/graceful-fs/graceful-fs.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/graceful-fs/polyfills.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/graceful-fs/legacy-streams.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/graceful-fs/clone.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/lodash.flattendeep/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/release-zalgo/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/release-zalgo/lib/Async.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/release-zalgo/lib/Sync.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/release-zalgo/lib/Thenable.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/release-zalgo/lib/constants.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/release-zalgo/lib/unwrapSync.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/es6-error/lib/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/find-cache-dir/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/commondir/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/pkg-dir/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/find-up/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/locate-path/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/p-locate/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/p-limit/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/p-try/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/path-exists/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/nyc/lib/fs-promises.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/glob/glob.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/fs.realpath/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/fs.realpath/old.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/minimatch/minimatch.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/brace-expansion/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/concat-map/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/balanced-match/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/inherits/inherits.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/path-is-absolute/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/glob/sync.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/glob/common.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/inflight/inflight.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/wrappy/wrappy.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/once/once.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/nyc/lib/hash.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-coverage/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-coverage/lib/file-coverage.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-coverage/lib/percent.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-coverage/lib/data-properties.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-coverage/lib/coverage-summary.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-coverage/lib/coverage-map.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-hook/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-hook/lib/hook.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/append-transform/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/default-require-extensions/js.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/strip-bom/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-processinfo/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/uuid/dist/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/uuid/dist/v1.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/uuid/dist/rng.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/uuid/dist/stringify.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/uuid/dist/validate.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/uuid/dist/regex.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/uuid/dist/v3.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/uuid/dist/v35.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/uuid/dist/parse.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/uuid/dist/md5.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/uuid/dist/v4.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/uuid/dist/v5.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/uuid/dist/sha1.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/uuid/dist/nil.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/uuid/dist/version.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/archy/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/cross-spawn/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/cross-spawn/lib/parse.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/cross-spawn/lib/util/resolveCommand.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/which/which.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/isexe/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/isexe/mode.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/path-key/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/cross-spawn/lib/util/escape.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/cross-spawn/lib/util/readShebang.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/shebang-command/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/shebang-regex/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/cross-spawn/lib/enoent.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/rimraf/rimraf.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/p-map/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/aggregate-error/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/indent-string/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/clean-stack/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/nyc/lib/source-maps.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/convert-source-map/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-source-maps/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-source-maps/lib/map-store.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/debug/src/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/debug/src/node.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/supports-color/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/has-flag/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/debug/src/common.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/ms/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/source-map/source-map.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/source-map/lib/source-map-generator.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/source-map/lib/base64-vlq.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/source-map/lib/base64.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/source-map/lib/util.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/source-map/lib/array-set.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/source-map/lib/mapping-list.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/source-map/lib/source-map-consumer.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/source-map/lib/binary-search.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/source-map/lib/quick-sort.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/source-map/lib/source-node.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-source-maps/lib/pathutils.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-source-maps/lib/transformer.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-source-maps/lib/mapped.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-source-maps/lib/get-mapping.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-source-maps/lib/transform-utils.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/test-exclude/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@istanbuljs/schema/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@istanbuljs/schema/default-exclude.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@istanbuljs/schema/default-extension.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/test-exclude/is-outside-dir.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/test-exclude/is-outside-dir-posix.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/get-package-type/index.cjs
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/get-package-type/async.cjs
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/get-package-type/is-node-modules.cjs
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/get-package-type/cache.cjs
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/get-package-type/sync.cjs
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/nyc/lib/instrumenters/istanbul.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-instrument/package.json
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/nyc/package.json
/Users/kirrg001/dev/instana/nyc-reproduce/server.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-instrument/dist/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-instrument/dist/instrumenter.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/transformation/file/file.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/react/isReactComponent.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/buildMatchMemberExpression.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/matchesPattern.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/generated/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/utils/shallowEqual.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/utils/deprecationWarning.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/react/isCompatTag.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/builders/react/buildChildren.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/utils/react/cleanJSXElementLiteralChild.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/builders/generated/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/builders/validateNode.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/validate.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/definitions/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/to-fast-properties/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/definitions/core.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/is.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/isType.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/isPlaceholderType.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/isValidIdentifier.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/helper-validator-identifier/lib/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/helper-validator-identifier/lib/identifier.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/helper-validator-identifier/lib/keyword.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/helper-string-parser/lib/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/constants/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/definitions/utils.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/definitions/flow.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/definitions/jsx.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/definitions/misc.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/definitions/placeholders.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/definitions/experimental.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/definitions/typescript.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/definitions/deprecated-aliases.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/asserts/assertNode.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/isNode.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/asserts/generated/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/builders/flow/createTypeAnnotationBasedOnTypeof.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/builders/flow/createFlowUnionType.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/modifications/flow/removeTypeDuplicates.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/builders/typescript/createTSUnionType.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/modifications/typescript/removeTypeDuplicates.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/builders/generated/uppercase.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/clone/cloneNode.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/clone/clone.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/clone/cloneDeep.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/clone/cloneDeepWithoutLoc.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/clone/cloneWithoutLoc.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/comments/addComment.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/comments/addComments.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/comments/inheritInnerComments.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/utils/inherit.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/comments/inheritLeadingComments.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/comments/inheritsComments.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/comments/inheritTrailingComments.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/comments/removeComments.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/constants/generated/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/converters/ensureBlock.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/converters/toBlock.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/converters/toBindingIdentifierName.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/converters/toIdentifier.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/converters/toComputedKey.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/converters/toExpression.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/converters/toKeyAlias.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/modifications/removePropertiesDeep.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/traverse/traverseFast.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/modifications/removeProperties.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/converters/toSequenceExpression.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/converters/gatherSequenceExpressions.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/retrievers/getBindingIdentifiers.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/converters/toStatement.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/converters/valueToNode.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/modifications/appendToMemberExpression.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/modifications/inherits.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/modifications/prependToMemberExpression.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/retrievers/getOuterBindingIdentifiers.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/traverse/traverse.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/isBinding.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/isBlockScoped.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/isLet.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/isImmutable.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/isNodesEquivalent.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/isReferenced.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/isScope.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/isSpecifierDefault.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/isValidES3Identifier.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/types/lib/validators/isVar.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/tools/build-external-helpers.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/files/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/files/package.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/files/utils.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/caching.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/gensync-utils/async.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/gensync/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/util.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/gensync-utils/fs.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/errors/config-error.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/errors/rewrite-stack-trace.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/files/configuration.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/helpers/config-api.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/files/module-types.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/transform-file.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/full.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/plugin.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/helpers/deep-array.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/item.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/config-descriptors.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/gensync-utils/functional.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/resolve-targets.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/config-chain.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/validation/options.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/validation/removed.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/validation/option-assertions.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/pattern-to-regex.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/printer.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/validation/plugins.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/partial.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/helpers/environment.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/transformation/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/transformation/plugin-pass.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/transformation/block-hoist-plugin.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/transformation/normalize-opts.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/transformation/normalize-file.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/parser/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/parser/util/missing-plugin-helper.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/transformation/util/clone-deep.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/transformation/file/generate.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/transformation/file/merge-map.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/files/import.cjs
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/files/plugins.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/config/files/import-meta-resolve.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/vendor/import-meta-resolve.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/transform.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/transform-ast.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/core/lib/parse.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-instrument/dist/visitor.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-instrument/dist/source-coverage.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-instrument/dist/constants.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/template/lib/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/template/lib/formatters.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/template/lib/builder.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/template/lib/options.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/template/lib/string.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/template/lib/parse.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/parser/lib/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/code-frame/lib/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/highlight/lib/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/js-tokens/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/chalk/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/escape-string-regexp/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/ansi-styles/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/color-convert/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/color-convert/conversions.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/color-name/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/color-convert/route.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/chalk/templates.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/template/lib/populate.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/template/lib/literal.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/istanbul-lib-instrument/dist/read-coverage.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/helper-compilation-targets/lib/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/browserslist/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/node-releases/data/processed/envs.json
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/caniuse-lite/dist/unpacker/agents.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/caniuse-lite/dist/unpacker/browsers.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/caniuse-lite/data/browsers.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/caniuse-lite/dist/unpacker/browserVersions.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/caniuse-lite/data/browserVersions.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/caniuse-lite/data/agents.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/node-releases/data/release-schedule/release-schedule.json
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/electron-to-chromium/versions.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/browserslist/error.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/browserslist/parse.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/browserslist/node.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/caniuse-lite/dist/unpacker/feature.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/caniuse-lite/dist/lib/statuses.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/caniuse-lite/dist/lib/supported.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/caniuse-lite/dist/unpacker/region.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/helper-validator-option/lib/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/helper-validator-option/lib/validator.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/helper-validator-option/lib/find-suggestion.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/compat-data/native-modules.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/compat-data/data/native-modules.json
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/lru-cache/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/yallist/yallist.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/yallist/iterator.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/helper-compilation-targets/lib/utils.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/helper-compilation-targets/lib/targets.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/helper-compilation-targets/lib/options.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/helper-compilation-targets/lib/pretty.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/helper-compilation-targets/lib/debug.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/helper-compilation-targets/lib/filter-items.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/compat-data/plugins.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/compat-data/data/plugins.json
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/visitors.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/path/lib/virtual-types.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/cache.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/traverse-node.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/context.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/path/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/scope/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/scope/lib/renamer.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/helper-split-export-declaration/lib/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/helper-environment-visitor/lib/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/scope/binding.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/globals/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/globals/globals.json
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/generator/lib/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/generator/lib/source-map.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@jridgewell/gen-mapping/dist/gen-mapping.umd.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@jridgewell/set-array/dist/set-array.umd.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@jridgewell/sourcemap-codec/dist/sourcemap-codec.umd.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@jridgewell/trace-mapping/dist/trace-mapping.umd.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@jridgewell/trace-mapping/node_modules/@jridgewell/sourcemap-codec/dist/sourcemap-codec.umd.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@jridgewell/resolve-uri/dist/resolve-uri.umd.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/generator/lib/printer.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/generator/lib/buffer.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/generator/lib/node/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/generator/lib/node/whitespace.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/generator/lib/node/parentheses.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/generator/lib/generators/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/generator/lib/generators/template-literals.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/generator/lib/generators/expressions.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/generator/lib/generators/statements.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/generator/lib/generators/classes.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/generator/lib/generators/methods.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/generator/lib/generators/modules.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/generator/lib/generators/types.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/jsesc/jsesc.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/generator/lib/generators/flow.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/generator/lib/generators/base.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/generator/lib/generators/jsx.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/generator/lib/generators/typescript.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/path/ancestry.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/path/inference/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/path/inference/inferers.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/path/inference/inferer-reference.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/path/inference/util.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/path/replacement.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/helper-hoist-variables/lib/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/path/evaluation.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/path/conversion.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/helper-function-name/lib/index.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/path/introspection.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/path/context.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/path/removal.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/path/lib/removal-hooks.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/path/modification.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/path/lib/hoister.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/path/family.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/path/comments.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/path/lib/virtual-types-validator.js
/Users/kirrg001/dev/instana/nyc-reproduce/node_modules/@babel/traverse/lib/hub.js
```

I don't have any other ideas other then cleaning up any required module from cache, which is an instrumentation for us. The cleaning should happen in a centralized place.

Reply:
Setting `useSpawnWrap` to true does not help.

Reply:
e.g. https://www.npmjs.com/package/c8 is not making use of node-preload.